content/en/docs/project-user-guide/storage/volumes.md

```diff
@@ -33,7 +33,7 @@ KubeSphere can get supported features of underlying storage plugin that called `
 The console display only supported features in `Volume Detail Page`.
```

Comment:
supported -> supports

Reply:
I mean `passive`

Reply:
->
The console only displays supported features in `Volume Detail Page`.

I think the word "display" is used as a "verb" here and that "supported" is used as an adjective.